test.ts

```diff
@@ -2037,26 +1976,25 @@ var TESTS: Test[] = [
    * Unnamed group prefix.
    */
   [
-    '/(apple-)?icon-:res(\\d+).png',
+    '\\/(apple-)?icon-:res(\\d+).png',
```

Comment:
It's surprising that this would require a double backslash instead of `'\/(apple-)...'`.

For example, `new RegExp('\/')` and `new RegExp('\\/')` are equivalent.

Reply:
Gotcha. This is just because of JavaScript strings. You need to escape the backslash because JavaScript interprets it as an escape character before the library does.

Reply:
Note that using a RegExp special character will demonstrate the difference (just slash doesn’t mean anything in RegExps).